pkg/arm/arm.go

```diff
@@ -43,16 +44,23 @@ func (*simpleGenerator) Generate(ctx context.Context, cs *acsapi.OpenShiftManage
 	if err != nil {
 		return nil, err
 	}
+
+	devModeFunc := func() bool {
+		return os.Getenv("DEVELOPMENT_MODE") != ""
```

Comment:
I'm on the fence about whether it makes sense to use the same env var for:
1- disabling LogAnalytics in at least most tests (permanent move; rationale: it's expensive)
2- enabling rotation for some dev/test? (temporary; rationale: getting more soak testing?)
I guess it's OK.

Clearly we want to minimise the number of these special cases over time.
Also would an env var like RUNNING_UNDER_TEST or something be a clearer name than DEVELOPMENT_MODE?

Reply:
`DEVELOPMENT_MODE` is more clear for me than `RUNNING_UNDER_TEST` :) so question if perception :)

Reply:
Yes, but it's not an environment variable that developers should really be using.